scenarios/centos-9/hci_ceph_backends.yml

```diff
@@ -24,26 +24,18 @@ cifmw_block_device_size: 20G
 
 cifmw_services_manila_enabled: true
 
-pre_tempest:
-  - name: Create manila resources
+pre_tests:
+  - name: 90 Create manila resources
     type: playbook
     source: manila_create_default_resources.yml
 
 cifmw_run_tests: true
 cifmw_tempest_container: openstack-tempest-all
-cifmw_tempest_default_groups: &test_operator_list
```

Comment:
Not working with test_operator, so each operator will provide its own include/exclude list.